tutorials/integration.md

```diff
@@ -150,31 +151,42 @@ import "@opengsn/gsn/contracts/forwarder/IForwarder.sol";
 import "@opengsn/gsn/contracts/BasePaymaster.sol";
 
 ```
+
```

Comment:
remove manual formatting. look at the github generated site first, and we'll touch-up the template if needed, not "shotgun" the entire source with newlines.

Reply:
Do you mean the &nbsp; ? When I have code and paragraphs interspaced there's a risk that people get confused which code goes with which paragraph. I don't think that can be solved in the template.